orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf

```diff
@@ -64,16 +64,19 @@ module "orc8r-app" {
   orc8r_db_user    = module.orc8r.orc8r_db_user
   orc8r_db_pass    = module.orc8r.orc8r_db_pass
 
-  # If you want to use your own container images or Helm charts, override the
-  # following relevant variables. These examples assume DockerHub for the
-  # container image registry and a GitHub repo for the charts repo.
-  # docker_registry = "registry.hub.docker.com/MY_USERNAME"
+  # Note that this can be any container registry provider -- the example below
+  # provides the URL format for Docker Hub, where the user and pass are your
+  # Docker Hub username and access token, respectively
+  docker_registry = "https://docker.artifactory.magmacore.org/artifactory/docker"
   # docker_user     = "MY_USERNAME"
   # docker_pass     = "MY_PASSWORD"
-  # helm_repo       = "https://raw.githubusercontent.com/MY_USERNAME/MY_REPONAME/master/"
+
+  # Note that this can be any Helm chart repo provider -- the example below
+  # provides the URL format for using a raw GitHub repo, where the user and
+  # pass are your GitHub username and access token, respectively
+  helm_repo = "https://docker.artifactory.magmacore.org/artifactory/helm-prod"
   # helm_user       = "MY_USERNAME"
   # helm_pass       = "MY_PASSWORD"
```

Comment:
We already set these defaults, so can just leave it commented-out as I have it. But if you think it reads better the way you have it, that's fine with me

Reply:
Rewrote the comments and uncommented credentials with empty string as defaulted in the variables file